syntax = "proto3";
// 解析数据的数据格式
// 消息通知
message ProxyMessage {
  enum ProxyStatus {
    // 已经使用
    USED = 0;
    // 可以使用
    INIT = 1;
    // 无法使用
    ERROR = 2;
  }
  // 唯一id
  string id = 1;
  // 创建时间戳
  string create_time = 2;
  ProxyStatus status = 3;
  // 使用时间
  string use_time = 4;
  // 代理ip值
  string ip = 5;
}
